Survey/Sources/Presentation/Modules/Home/Views/PersonalMenuView.swift

```diff
@@ -0,0 +1,148 @@
+                            }
+                        ),
+                        secondaryButton: .default(
+                            Text(AssetLocalization.commonCancelText()),
+                            action: {
+                                showAlert = false
+                            }
+                        )
+                    )
+                }
+                .padding()
+                .frame(width: 240.0, height: UIScreen.main.bounds.height)
+                .background(Color.grayScale)
+                .opacity(0.9)
+        }
+        .edgesIgnoringSafeArea(.all)
+    }
+
+    private func setUpAvatarButton() -> some View {
+        Button(action: {}, label: {
+            AsyncImage(
+                url: URL(string: user.avatarUrl),
+                transaction: Transaction(animation: .easeIn(duration: 1.0))
+            ) { phase in
+                switch phase {
+                case .empty:
+                    ProgressView().hidden()
+                case let .success(image):
+                    image
+                        .resizable()
+                        .scaledToFill()
+                        .clipShape(Circle())
+                case .failure:
+                    Image(systemName: "person.2.circle")
+                        .resizable()
+                        .scaledToFill()
+                        .clipShape(Circle())
+                @unknown default:
+                    EmptyView()
+                }
+            }
+            .frame(width: 36.0, height: 36.0)
+        })
+        .padding()
+    }
+
+    private func setUpComponents() -> some View {
+        VStack {
+            HStack {
+                Text(user.name)
+                    .font(.largeTitle)
+                    .foregroundColor(.white)
+                Spacer()
+                setUpAvatarButton()
+            }
+            .padding(.top, 10.0)
+            Divider()
+                .background(.white)
+            HStack {
+                Button(AssetLocalization.userLogoutButtonText()) {
+                    showAlert.toggle()
+                }
+                .font(.regularBody)
+                .foregroundColor(.white)
+                .opacity(0.5)
+                Spacer()
+            }
+            .padding(.top, 20.0)
+
+            Spacer()
+        }
+    }
+
+    private func setUpLeftView() -> some View {
+        ZStack {
+            Color.black.opacity(0.0)
+            Rectangle()
+                .frame(
+                    width: UIScreen.main.bounds.width - 240.0,
+                    height: UIScreen.main.bounds.height
+                )
+                .blendMode(.destinationOut)
+                .onTapGesture {
+                    withAnimation(.easeIn) {
+                        showPersonalMenu = false
+                    }
+                }
+        }
+        .compositingGroup()
+    }
+
+    private func setUpVersion() -> some View {
+        VStack(alignment: .leading) {
```

Comment:
Not sure where to see this component in the pow?

Reply:
- I just noticed the video is before i added version text 😄, just updated POW.

<img width=400 src=https://user-images.githubusercontent.com/25881847/189581239-1b6d9f00-151c-419b-969d-3a2775d7e6c4.png>